### DISCUSSION TITLE
Term Frequency adjustment

### DISCUSSION THREAD

**Discussion by msiemionCalistapw** [moj-analytical-services/splink#1240] [Q&A]
Is there a way to have less of a term frequency adjustment? I have seen in some examples a term frequency adjustment weight given to the comparison but based on the way my comparisons are written I'm not sure how to write in a term frequency adjustment weight. 

My code: 
Location_comparison = cl.levenshtein_at_thresholds("Location", distance_threshold_or_thresholds=5, term_frequency_adjustments= True)

**Comment by RossKen** [comment]
Hey @msiemionCalistapw, you can currently only add a term frequency adjustment weight when specifying your comparison as a dictionary, it is not a parameter in the `cl` or `cll` libraries. We are just about to release a topic guide on term frequency adjustments, you can see the relevant part for fuzzy matching (including tf adjustment weight) [in the open PR](https://github.com/moj-analytical-services/splink/pull/1214/files#diff-8b934ffd402d9263b50281803e578287cb1399f20ce494a9f2de444cb738d550R188-R206).

**Reply by msiemion** [reply]
Great, thank you for the help!
